.github/workflows/nightly-build.yml

```diff
@@ -73,35 +73,40 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: oneapi-src/oneDAL
+      - name: Set MKL_FPK_GPU_VERSION_LINE
+        shell: bash
+        run: |
+          echo "MKL_FPK_GPU_VERSION_LINE=2024.0.0" >> "$GITHUB_ENV"
       - name: Install DPC++
         shell: cmd
         run: |
-            call .\.ci\scripts\install_windows.bat ${{ env.WINDOWS_BASEKIT_URL }} ${{ env.WINDOWS_ALL_COMPONENTS }}
+            call .\.ci\scripts\install_windows.bat ${{ env.WINDOWS_BASEKIT_URL }} ${{ env.WINDOWS_ALL_COMPONENTS }} .\oneapi\
```

Comment:
can you also as part of this work add variable for control version instead of hardcoding it
https://github.com/oneapi-src/oneDAL/pull/2920/files#diff-2ee59052448a421cff47b6ee98f786b57f3012d9f9132d37b9bde685a6266166R28

Reply:
usually a trailing oneapi is necessary, I'm not sure this is necessary